pisoni.gemspec

```diff
@@ -18,12 +18,13 @@ Gem::Specification.new do |s|
   s.description = 'Client for the Apisonator internal API for model data.'
   s.license     = 'Apache-2.0'
 
-  s.add_dependency 'faraday', '>= 0.9.1'
-  s.add_dependency 'json', '>= 1.8.1'
-  s.add_dependency 'injectedlogger', '>= 0.0.13'
-  s.add_dependency 'net-http-persistent'
+  # faraday v2.9.0 removes support for Ruby 2.7, see https://github.com/lostisland/faraday/releases/tag/v2.9.0
+  s.add_runtime_dependency 'faraday', '~> 2.0', '<= 2.9'
+  s.add_runtime_dependency 'json', '~> 2.7', '>= 2.7.1'
+  s.add_runtime_dependency 'injectedlogger', '0.0.13'
+  s.add_runtime_dependency 'faraday-net_http_persistent', '~> 2.1'
 
-  s.add_development_dependency 'rake'
+  s.add_development_dependency 'rake', '~> 13.1'
```

Comment:
I don't think we care about rake to limit it to version 13. I'd only prefer that we leave it as it was or don't set upper bound.

Reply:
We don't, but I don't like the warnings shown when building the gem:
```
WARNING:  open-ended dependency on rake (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'
```

Reply:
ok, limit it to version 100 then? :)

Reply:
Dunno... Looking at the versions history of `rake` it doesn't seem to me like a project that bumps the major version too often. And if/when it does, maybe there would be a good reason for that - e.g. breaking changes, so probably we would appreciate if our builds don't start failing for some reason (reason being - that our Rakefiles are not compatible anymore).
I'd keep it like this.

Reply:
If a version is required to remove the warning, then better leave it at `~> 13.1`.

`~> 100` is supper ugly

Reply:
Ok, good. But don't complain to me when we are still using rake 13 after 15 years! :sunglasses:

Reply:
Deal @akostadinov :laughing: